releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,16 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.2.0 - February 29, 2024
+
+### Enhancements
+
+* Supported declaring feature flags with [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json)
```

Comment:
Sorry for the nit, but the FM library doesn't declare feature flags. It reads feature flags declared by someone else. Does this work?

Added support for feature flags defined using [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v1.0.0.schema.json).

Reply:
Sounds good.